CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
 <!-- Keep a Changelog guide -> https://keepachangelog.com -->
 
 # TestSpark Changelog
-
 ## Unreleased
+## 0.3.0
+### Fixed
+- Multiple bugs in the UI
+- Multiple bugs in the LLM-based test generation algorithm
+### Changed
+- Major refactoring: removed most of the global services
+### Added
+- Full LLM-based Kotlin test generation for line, method/function, and class
+- Integration with HuggingFace
```

Comment:
Do you want to specify that issue we found yesterday about the K2 frontend of the compiler?